enterprise_gateway/services/processproxies/yarn.py

```diff
@@ -159,11 +157,20 @@ def confirm_yarn_queue_availability(self, **kwargs):
 
         self.start_time = RemoteProcessProxy.get_current_time()
         self.candidate_queue = self.resource_mgr.cluster_scheduler_queue(candidate_queue_name)
+
+        if self.candidate_queue is None:
+            self.log.debug("Queue: {} does not exist in the cluster".format(candidate_queue_name))
```

Comment:
Since this is requested functionality, by virtue of a non-null `KERNEL_QUEUE` value, I think we should log this as a warning and add " - availability check will not be performed."

Reply:
Commited the changes.